packages/example/src/pages/components/markdown.mdx

```diff
@@ -35,6 +35,12 @@ Combined emphasis with **asterisks and _underscores_**.
 Strikethrough uses two tildes. ~~Scratch this.~~
 ```
 
+## Headers
```

Comment:
Adds documentation around recommended IA and notes the utility of h1s for starting pages.